ui.R

```diff
@@ -87,6 +87,73 @@ navbarPage("Power Implications for Reduced Sample Size",
                       )
                     )
            ),
+           
+           # ***************************************************
+           # GSD start
+           # ***************************************************
+           tabPanel("GSD",
+                    sidebarLayout(
+                      sidebarPanel(
+                        selectInput(inputId = "designSelect", label="Type of GSD", 

```

Comment:
Currently, this may give the impression that we design a GSD in the first place. But in fact, we add an interim analysis but don't change other design features (e.g., total sample size). I'd suggest that we introduce this input after "tauGSD".

Reply:
It would also be helpful to add a tooltip to "designSelect" to explain how this GSD is implemented. For example,

                        bsTooltip("designSelect",
                                  "Implement an interim analysis with data available and a final analysis when all data are collected",
                                  "right",
                                  trigger = "hover", options = list(container = "body")
                        )

Reply:
Thanks for the suggestion. I added this now.

Reply:
> Currently, this may give the impression that we design a GSD in the first place. But in fact, we add an interim analysis but don't change other design features (e.g., total sample size). I'd suggest that we introduce this input after "tauGSD".
> 
> Also we can use this place to define the acronym GSD.

I am wondering about your thought about introducing the "designSelect" after "tauGSD". The current order may give the impression that we are designing a gsd from the beginning.